enumerable.rb

```diff
@@ -0,0 +1,15 @@
+module Enumerable
+  def all?
+    each { |n| return false unless yield(n) }
+  end
```

Comment:
- [ ] if in case of the provided example 
```
list = MyList.new(1, 2, 3, 4)
list.all? {|e| e < 5}
``` 
It is not returning anything, in order to fix it the `all?` method should return `true` at the end.